legal/satzung.md

```diff
@@ -0,0 +1,123 @@
+# Satzung
+
+
+## Name, Sitz, Geschäftsjahr
+
+1.   Der Verein trägt den Namen *open energy modeling foundation* (oemof)
+2.   Sitz des Vereins ist $(SITZ).
+3.   Das Geschäftsjahr ist das Kalenderjahr.
+
+
+## Zweck und Ziele
+
+1. Der Verein verfolgt ausschließlich und unmittelbar gemeinnützige Zwecke im Sinne des Abschnitts „Steuerbegünstigte Zwecke“ der Abgabenordnung.
+2. Zweck des Vereins ist die Förderung von Wissenschaft und Forschung.
+    Dieser Zweck wird verwirklicht durch
+    1. Bereitstellung wissenschaftlicher Software unter OSI-anerkannten Open-Source-Lizenzen,
+    2. Unterstützung bei der Weiterentwicklung, Wartung und Dokumentation ebendieser Software, und
+    3. $(ZWECK)
+
+
+## Selbstlosigkeit
+
+1. Der Verein ist selbstlos tätig. Er verfolgt nicht in erster Linie eigenwirtschaftliche Zwecke.
+2. Die Mittel des Vereins dürfen nur für seine satzungsmäßigen Zwecke verwendet werden.
+    Die Mitglieder dürfen keine Gewinnanteile und in ihrer Eigenschaft als Mitglieder auch keine sonstigen Zuwendungen aus Mitteln des Vereins erhalten.
+3. Es darf keine Person durch Ausgaben, die dem Zweck des Vereins fremd sind, oder durch unverhältnismäßig hohe Vergünstigungen begünstigt werden.
+
+
+## Organe des Vereins
+
+1. Die Organe des Vereins sind
+    1. die Mitgliederversammlung und
+    2. der Vorstand.
+2. Der Vorstand besteht aus drei Mitgliedern,
+    1. der Sprecher*in,
+    2. der Schriftführer*in und
+    3. der Kassenwart*in.
+    Diese dürfen nicht beim selben Arbeitgeber beschäftigt sein.
```

Comment:
```suggestion
    Jedes Vorstandsmitglied muss bei einem anderen Arbeitgeber beschäftigt sein.
```
Könnte sonst missverständlich sein, da man annehmen könnte, dass auch zwei Arbeitgeber ausreichen.

Reply:
So hatte ich es auch mal überlegt, doch könnte diese Formulierung so gelesen werdne, dass sie Selbständige ausschließt.

Reply:
Vielleicht dann besser: "Diese dürfen jeweils nicht beim selben Arbeitgeber beschäftigt sein."? Ist vielleicht auch nur eine Kleinigkeit. Ich hatte es auch richtig verstanden, aber meine Kollegin hatte es beim ersten Lesen missverstanden.

Reply:
Und könnte dann nun auch eine Institut Vorstand werden? (Ich bin für nein - aber das müsste man dann noch hinzufügen, oder?) Denn falls ja, wo wäre eine Institut beschäftigt? ;)

Reply:
Hmm, gute Frage. Vielleicht kann man es so machen, dass entweder natürliche Personen oder der/die jeweilige Vertreter:in des Instituts in den Vorstand gewählt werden kann.

Reply:
Ich sehe nur natürliche Personen als Vorstandsmitglieder. Wie wäre es mit folgenden Unterpunkten? 

    3. Die Vorstandsmitglieder müssen voll geschäftsfähige, natürlich Personen sein.
    4. Jeweils zwei Vorstandsmitglieder dürfen nicht beim selben Arbeitgeber beschäftigt sein.